app/main/menu.ts

```diff
@@ -211,7 +211,7 @@ class AppMenu {
 	}
 
 	getWindowSubmenu(tabs, activeTabIndex) {
-		const initialSubmenu = [{
+		const initialSubmenu: any[] = [{
```

Comment:
We should declare a interface for this. Using a interface would do couple of this will make sure any function consuming this will get exactly what it expected.

(The `noImplicitAny` tsconfig.json option will make sure we can't do this.)

Reply:
I changed noImplicitAny to ```true``` but still could do this.